exasol/analytics/schema/table.py

```diff
@@ -17,10 +17,8 @@ def __init__(self, name: TableName, columns: List[Column]):
     @property
     def create_statement(self):
         columns = ",\n  ".join(c.for_create for c in self.columns)
-        return cleandoc(
-            f"""
-            CREATE TABLE IF NOT EXISTS {self.name.fully_qualified} (
-              {columns}
-            )
-            """
+        return (
+            f"CREATE TABLE IF NOT EXISTS {self.name.fully_qualified} (\n"
+            f"  {columns}\n"
+            ")"
```

Comment:
I would suggest, to  concat the columns only with `,\n` and use [textwrap.indent](https://docs.python.org/3/library/textwrap.html#textwrap.indent)

Reply:
Ah - thanks. Great. 
See next push.